include/ck/tensor_operation/gpu/block/blockwise_gemm_xdlops.hpp

```diff
@@ -151,6 +151,27 @@ struct BlockwiseGemmXdlops_k0mk1_k0nk1_m0n0m1n1m2m3m4n2_v1
         return make_tuple(c_thread_m, c_thread_n);
     }
 
+    template <index_t m0, index_t n0, index_t xdlops_i, index_t blk_i>
+    __device__ static auto
```

Comment:
why not just use ```indez_t``` instead of  ```Number``` for the function argument, to redue some templating?

Reply:
Fair point. This was before we found concrete evidence about the long build time. We need to discourage using `Number<int>` going forward.